components/OpenEmailProviderButton.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StyledButton from './StyledButton';
+import { FormattedMessage } from 'react-intl';
+
+function OpenEmailProviderButton({ email }) {
+  const gmailregex = /@gmail\.com$/.test(email);
+  const hotmailregex = /@outlook\.com$/.test(email);
+  const gmaillink =
+    'https://mail.google.com/mail/u/2/#advanced-search/subject=Open+Collective%3A+Login&amp;subset=all&amp;within=1d';
+  const hotmaillink = 'https://outlook.live.com/mail/inbox';
+  const btntext = gmailregex ? 'Gmail' : hotmailregex ? 'HotMail' : 'Inbox';
+  const link = gmailregex ? gmaillink : hotmailregex ? hotmaillink : '';
+  const emailProvider = gmailregex || hotmailregex ? true : false;
+
+  return emailProvider ? (
+    <div>
+      <a id="emailprovider" href={link}>
```

Comment:
As per https://docs.opencollective.com/help/developers/testing-with-cypress let's use `data-cy` rather than an `id`

> Use `data-cy` to target DOM elements
> Don’t target elements based on CSS attributes such as: id, class, tag Add data-* attributes to make it easy to target elements

```suggestion
      <a data-cy="open-inbox-link" href={link}>
```